calcom/cal.com
# fix: bookings for guests added by organizer get cancelled when rescheduled

## What does this PR do?



- Fixes #17813 (GitHub issue number)
- Fixes CAL-XXXX (Linear issue number - should be visible at the bottom of the GitHub issue description)



[Screencast from 25-11-24 12:25:13 AM IST.webm](https://github.com/user-attachments/assets/2e67a49a-7fd0-4f92-8201-feefda492e17)


## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

1. Create a booking on team pro example account.
2. Login to team pro example account and Go to /bookings/incoming page.
3. Add guests to that booking.
4. Now Reschedule the booking.
5. Notice that the guests that were just added (by team pro example) are now included in the rescheduled booking.
